content/nextflow-on-aws-batch/35_nextflow201/11_job_queue.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Create Job Queues"
+chapter: false
+weight: 11
+---
+
+## Create Job Queue
+
+Let us created a queue via the console.
```

Comment:
create not created

Reply:
done

Reply:
Evidence shows that you lied or didn't commit :)

Reply:
now for real